src/mpid/ch4/shm/posix/eager/src/posix_eager_impl.c

```diff
@@ -0,0 +1,24 @@
+/*
+ *  (C) 2020 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ *
+ */
+
+#ifndef POSIX_EAGER_INLINE
+#ifndef POSIX_EAGER_DISABLE_INLINES
+
```

Comment:
I do not think the `#ifndef POSIX_EAGER_INLINE` wrapper is needed for .c

Reply:
Then the code would be always compiled? I am not even sure why we have the `POSIX_EAGER_DISABLE_INLINES` in `posix_eager_impl.c` (and the netmod as well) as the define is done in a separate compilation unit (i.e. `fbox/func_table.c`).

Reply:
As long as posix/eager is built, this part has to be always compiled, right? I think we always build posix/eager.

It looks like regular wrapper for header file, but should not be used for .c. Can you please test without the macros?

Reply:
I think `POSIX_EAGER_DISABLE_INLINES` can be safely removed (in fact it is never defined in this compilation unit), `POSIX_EAGER_INLINE` should be left or else duplicated symbols are found when compiling with inlining enabled (I have tested it).